packages/@react-spectrum/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -141,17 +141,19 @@ function Breadcrumbs<T>(props: SpectrumBreadcrumbsProps<T>, ref: DOMRef) {
 
     let menuItem = (
       <BreadcrumbItem key="menu">
-        <MenuTrigger>
-          <ActionButton
-            aria-label="…"
-            isQuiet
-            isDisabled={isDisabled}>
-            <FolderBreadcrumb />
-          </ActionButton>
-          <Menu selectionMode="single" selectedKeys={[selectedKey]} onAction={onMenuAction}>
-            {childArray}
-          </Menu>
-        </MenuTrigger>
+        <div>
+          <MenuTrigger>
+            <ActionButton
+              aria-label="…"
+              isQuiet
+              isDisabled={isDisabled}>
+              <FolderBreadcrumb />
+            </ActionButton>
+            <Menu selectionMode="single" selectedKeys={[selectedKey]} onAction={onMenuAction}>
+              {childArray}
+            </Menu>
+          </MenuTrigger>
+        </div>
```

Comment:
Hrm, I guess this is fine as a stopgap but yeah it would be nice to get the BreadcrumbItem props sent to the ActionButton instead (maybe through a Provider of some sort? PressResponder?).

Reply:
Well, the issue is https://github.com/adobe-private/react-spectrum-v3/blob/master/packages/%40react-spectrum/breadcrumbs/src/BreadcrumbItem.tsx#L36

Also, ActionButton doesn't accept slots currently, don't know if we'd really want to change that.

Otherwise we could pass through a few props from menutrigger to the trigger, but that's something we're actively avoiding I think

Reply:
Yep, so I was hoping that wrapping the MenuTrigger in a PressResponder would successfully pass down the ref + className + itemProps that BreadcrumbItem clones onto the top element (which would be the PressResponder). It kinda works (the BreadcrumbItem props from the cloneElement appear as the prev context, and the breadcrumb item class name makes it into the ActionButton's `buttonProps` via  PressResponderContext -> `usePress` -> `useButton` -> ActionButton) but the className gets overwritten by the className in ActionButton since it wasn't shuttled in via the ActionButton props.

The DOMPropsResponder (https://github.com/adobe-private/react-spectrum-v3/blob/f0f104dbd5e2e70b954e50308f64ec8b2e61ab7e/packages/%40react-aria/interactions/src/DOMPropsResponder.tsx) was meant for situations like this, but I'm not sure if we even want DOMPropsResponder to be a thing anymore. Slots would work but feels kinda bad cuz it feels like it begins to shy away from the original usage of SlotProvider and more into DOMPropsResponder territory.

Reply:
can MenuTrigger forward the ref to the button?

Reply:
sure, but what about the classname?

also, thought we were avoiding passing things through to the children because we expose them already, so if someone put a ref on the button and on the menutrigger... what happens?

Reply:
what class name?

I guess we'd need to merge the refs...

Reply:
spectrum-Breadcrumbs-itemLink